### ISSUE TITLE
releasing multiple opam packages from a single repository

### ISSUE THREAD

**Issue by hannesm** [ocamllabs/dune-release#442]
first, thanks for your work on dune-release to make releases of opam packages that use dune so easy.

with dune-release 1.6.1 the synopsis is taken from a random opam file. e.g. https://github.com/ocaml/opam-repository/pull/21158 uses the `dnssec.opam` (where my expectation would have been to use `dns.opam` -- the one in dune-project).

**Comment by aryx** [comment]
Is there any news on this topic? Can we use dune-release with repository containing multiple opam files?

**Comment by Leonidas-from-XIV** [comment]
> Can we use dune-release with repository containing multiple opam files?

Yes, you can ([ocaml-zmq](https://github.com/issuu/ocaml-zmq) does that for example).

But the semantics are not necessarily solid as @hannesm notes. It should definitely prioritize getting the synopsis for the PR from the package in the `(name)` field in `dune-project` if available.
